### ISSUE TITLE
Use Shadow Cost for EV Charger Deployment Lever

### ISSUE THREAD

**Issue by jrissman** [EnergyInnovation/eps-us#208]
From Robbie:

Yes, I think this lever requires a revisit. Right now, it simply causes a linear increase in EV adoption, which is not realistic, particularly at higher levels. For example when we put on the infrastructure bill charging spending, it caused a 5-10% increase in sales, even though we already had a $12,500 incentive. Seems very unlikely.

There is a better way to do this which relies on other data we already use to calculate the shadow costs. That report includes a few things, the two main features of which are costs related to limited range and costs related to the lack of charging density. 

We actually used the report to estimate the perceived cost of current lack of charging infrastructure and the perceived decreased in cost with an increase in chargers in the infrastructure bill. This is nice because it leverages all the existing logit structure in the model and already yields decreasing marginal returns once the sales share gets high. I think it’s actually the proper way to represent how charging infrastructure can increase sales.

If we were to program this, here’s what I’d consider:

1)	There’s a few key pieces of data needed: BAU chargers/capita, BAU gas pumps/capita, and the curve information that relates the implicit cost per unit lower charger density.
2)	Then in the model we add the structure to approximate the costs using the curve information (kind of flexibility the curtailment coefficients we use). The model then calculates the implicit shadow price from charger availability and can also calculate the decreased shadow price when EV chargers increase from policy (or in BAU). 
3)	We could do something similar for range, since that’s the other part of the shadow price. In that case, the calculations of shadow prices would be entirely endogenous. We could remove the existing lever for range anxiety and instead tie the shadow price for chargers to the charger density shadow price and the range shadow price to the range of vehicles (maybe we add some type of new policy that increases the range… maybe R&D or something?)

**Comment by jrissman** [comment]
From Megan:

I support moving toward something like Robbie laid out below. For the current BBB modeling we’re working on, we did something similar by calculating the number of additional chargers that would be built based on the level of funding and our weighted average charger cost calculated in trans/EVCC. We then used that number to update the portion of the shadow cost calculation that relies on the ratio of EVSE ports to gas pumps. Our source data has the following data points, and we used Excel to find the logarithmic trendline to interpolate between points. But ideally, we should also find data on BAU charger deployment so that we can accurately factor that into our calculation as well.

![graph](https://user-images.githubusercontent.com/7120106/146631138-e9d2ab36-b6b5-4c46-bcbb-1bc101f60f5b.png)
